src/ngraph/runtime/cpu/cpu_tensor_view.cpp

```diff
@@ -36,6 +36,7 @@ const size_t runtime::cpu::CPUTensorView::BufferAlignment = 64;
 
 runtime::cpu::CPUTensorView::CPUTensorView(const ngraph::element::Type& element_type,
                                            const Shape& shape,
+                                           void* mem_handle,
```

Comment:
would it be better to add new parameters to the end of the parameter list? This parameter should have a default value so all existing calls are unchanged. Only calls making use of memory handle would need the extra argument.

Reply:
@rkimballn1 existing calls continue to work, we just added new api. I was thinking of modifying existing api with default param, but default parameters can add additional api user awareness, and also in this case we want to explicitly force users to specify a memory_pointer (no defaults used). existing code continued to work. i am ok changing, let me know will modify as needed.